static/js/chart/draw.ts

```diff
@@ -853,8 +854,12 @@ function drawGroupBarChart(
     return;
   }
 
+  // clear old chart to redraw over
+  const container = d3.select(containerElement);
+  container.selectAll("*").remove();
+
   const svg = d3
-    .select("#" + id)
+    .select(containerElement)
```

Comment:
nit: can replace d3.select(containerElement) with `container` that you defined in line 858

Reply:
nice catch! Updated.